en-us/install-instructions.md

```diff
@@ -89,6 +89,45 @@ For macOS or Linux [you need Java](https://adoptium.net/) to run the Fabric Inst
 
 A simple installer for vanilla launcher [is coming soon](https://github.com/Fabulously-Optimized/fabulously-optimized/issues/110)!
 
+### Pojav Launcher (Android)
+
+**This process is lengthy, may not work or may not improve performance for you. You've been warned.**
+
+All the following instructions should be ran on your high-end Android device.
+
+1. [Download Pojav Launcher](https://play.google.com/store/apps/details?id=net.kdt.pojavlaunch)
+   * Do not fall into scam websites. The only official site for it is https://github.com/PojavLauncherTeam/PojavLauncher
+2. [Download Material Files](https://play.google.com/store/apps/details?id=me.zhanghai.android.files)
+   * This is the file manager we'll be using for this tutorial. You could use others too, but instructions may vary.
+3. [Download Fabric installer](https://fabricmc.net/use/installer/) (jar version)
```

Comment:
```suggestion
3. [Download the Fabric installer](https://fabricmc.net/use/installer/) (as a `.jar` file)
```

Reply:
well there is an exe version and a jar version... 
not a "list of files"